src/Perlang.ConsoleApp/Program.cs

```diff
@@ -85,14 +85,32 @@ public static int MainWithCustomConsole(string[] args, IConsole console)
             var detailedVersionOption = new Option("-V", "Show detailed version information");
             var evalOption = new Option<string>("-e", "Executes a single-line script") { AllowMultipleArgumentsPerToken = false, ArgumentHelpName = "script" };
             var printOption = new Option<string>("-p", "Parse a single-line script and output a human-readable version of the AST") { ArgumentHelpName = "script" };
+            var noWarnAsErrorOption = new Option<string>("-Wno-error", "Treats specified warning as a warning instead of an error.") { ArgumentHelpName = "error" };
+
+            var disabledWarningsAsErrorsList = new List<WarningType>();
```

Comment:
(Silly `List` suffix to avoid name clash with non-static field. It's a shame the C# compiler uses the same scope for these, since a _static method cannot access the fields anyway_. Let's try to do better in Perlang when we implement support for `static` methods. :sunglasses:)